pkg/storage/sst_writer.go

```diff
@@ -26,7 +26,6 @@ import (
 // SSTWriter writes SSTables.
 type SSTWriter struct {
 	fw *sstable.Writer
-	f  io.Writer
```

Comment:
out of curiosity, is this just code cleanup?

Reply:
Yeah, I can't remember why I did this here but I got annoyed clicking around while hunting down the OOM, only to realize we never used it.